README.md

```diff
@@ -382,6 +382,43 @@ const engine = await firebolt.resourceManager.engine.getByName("engine_name")
 await engine.stop()
 ```
 
+<a id="create-engine"></a>
+##### Engine create
+
+Creates an engine.
+
+```typescript
+import { Firebolt } from 'firebolt-sdk'
+const firebolt = Firebolt();
+await firebolt.connect(connectionOptions);
+const engine = await firebolt.resourceManager.engine.create("engine_name");
+```
+
+<a id="attach-to-database"></a>
+##### Attach to database
+
+Attaches an engine to a database.
+
+```typescript
+import { Firebolt } from 'firebolt-sdk'
+const firebolt = Firebolt();
+await firebolt.connect(connectionOptions);
+const engine = await firebolt.resourceManager.engine.attach_to_database("engine_name", "database_name");
```

Comment:
I suggest to change all `snake_case` to `camelCase` names as it is follows javascript coding conventions